.github/workflows/js_tests.yml

```diff
@@ -0,0 +1,34 @@
+name: JavaScript
+on:
+  pull_request:
+    paths:
+      - 'webpack/**'
+      - 'package.json'
+      - '.github/workflows/js_tests.yml'
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        node-version:
+          - 10
+          - 12
```

Comment:
Would be nice to add `14` as well

Reply:
is it out already? :open_mouth:  added :+1: